app/models/manageiq/providers/storage_manager.rb

```diff
@@ -24,5 +25,20 @@ class << model_name
       define_method(:route_key) { "ems_storages" }
       define_method(:singular_route_key) { "ems_storage" }
     end
+
+    def model_feature_for_action(action)
```

Comment:
I think this belongs in `ui-classic`, e.g. https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/ems_network_controller.rb#L25-L30
cc @skateman @himdel